### PR TITLE
3.2 626

### PR DESCRIPTION
**Pull request by dodie**
Change array pattern to be handled as a simple separator character rather than a regular expression.

### CODE REVIEW

File: src/main/java/apoc/load/LoadCsv.java [hunk 1]
```diff
@@ -90,7 +90,7 @@ public Object convert(String value) {
         }
 
         private Object convertArray(String value) {
-            String[] values = value.split(arrayPattern);
+            String[] values = value.split(Pattern.quote(arrayPattern));
```
Comment (jexp) [inline src/main/java/apoc/load/LoadCsv.java]: pattern.split(value)

**Comment by jexp** [comment]
Better rebase the 3.2 branch than MERGE

File: src/main/java/apoc/load/LoadCsv.java (hunk 1 above)
Comment (dodie) [reply src/main/java/apoc/load/LoadCsv.java]: Ok, I change that.

**Comment by dodie** [comment]
Ok, I create a new PR with clean history.

**Pull request closed without merging**